### DISCUSSION TITLE
environment var can’t be change

### DISCUSSION THREAD

**Discussion by mesu0815** [dani-garcia/vaultwarden#4495] [General]
Hello, I have been using the Conatiner for a long time, now I wanted to make a few adjustments to the environment, but whenever I restart the container via Portainer, the old environment is back and the additional ones are gone.
I have to mention, I once had a config.json, I wanted to replace it and do everything in the environment, I renamed the config.json. 
For example, I also set an ADMIN_TOKEN with argon2, when I update the container the token is different and the log says :

NOTICE] You are using a plain text `ADMIN_TOKEN` which is insecure.
Please generate a secure Argon2 PHC string by using `vaultwarden hash` or `argon2`.
See: https://github.com/dani-garcia/vaultwarden/wiki/Enabling-admin-page#secure-the-admin_token

I'm honestly a bit at a loss.
Oh well, everything else works as it should, webapp etc. works

**Comment by stefan0xC** [comment]
You need to recreate the container as the environment variables are bound to the container.
Cf. https://github.com/dani-garcia/vaultwarden/wiki/Configuration-overview
